### DISCUSSION TITLE
Aeon labs DSB09 missing association groups 1-3

### DISCUSSION THREAD

**Discussion by TheKorn2** [zwave-js/zwave-js#6682] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

Aeon Labs DBS09 has a brazillion configuration options on reports to send to association groups 1-3.  However, groups 1-3 are not defined, so you can't add any nodes to be sent the reports.

1. Add a DSB09.
2. Click on groups
3. See lifeline group
4. Hit add
5. no groups but lifeline available.

### Software versions

zwave-js-ui: 9.8.2.c09b638
zwave-js: 12.4.3

Not using home assistant, but am using zwave-js-ui.

### Device information

Manufacturer: Aeon Labs
Model name: DSB09
Node ID: 125

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-02-20.zip](https://github.com/zwave-js/node-zwave-js/files/14353475/zwavejs_2024-02-20.zip)

**Comment by AlCalzone** [comment, marked as answer]
From what I gathered, association groups 2 and 3 do not actually exist. This is reported by the device and mentioned in the device manual.
The config parameters refer to "report groups" you can configure (which reports: params 101-103; how often: 111-113)

All reports are sent to association group 1 (the lifeline)

We do refer to association groups 2 and 3 in the device config file, which is wrong.
I've opened https://github.com/zwave-js/node-zwave-js/issues/6684 to keep track of this.

**Reply by AlCalzone** [reply]
Issue solved

**Comment by TheKorn2** [comment]
I am going to throw on logging for an example of this device and for the zw095 and let them run for a few hours.  I do not believe that meter reports are being sent to the lifeline. (Logging will tell the tale, obviously!)

I have them both configured for timed reports every ten minutes, so should have plenty of data.  Right now the zw095 hasn't updated since midnight (it's almost 6AM).

**Reply by AlCalzone** [reply]
Make sure to show the current configuration values too.

**Reply by TheKorn2** [reply]
@AlCalzone Well looks like the root cause of the zw095 not reporting was a setup snafu. Re-checking the Ps and Qs revealed the controller was never added to the lifeline group. Once I added it, meter reports are now flowing in.

Taking a second look at the dsb09s, they are reporting as configured. 

Apologies for the fire drill!
